pyramid_okta/security/authentication.py

```diff
@@ -75,8 +76,16 @@ def unauthenticated_userid(self, request):
         :param request: <pyramid.request>
         :return: user_id
         """
-        credentials = utils.get_credentials(request)
-        if credentials is None:
-            return None
-        user_id = credentials['client_id']
-        return user_id
+        if 'id' in request.session:
+            resp = utils.validate_session(request.session['id'])
```

Comment:
so only use the 'id' in request.session, otherwise, return None?  That'd work for me.